pipeline/stop_places/sql/stop_places.sql

```diff
@@ -555,28 +556,45 @@ $$
 LANGUAGE plpgsql IMMUTABLE STRICT;
 
 
+/* 
+ * Create a function, that estimates the duration based on the length of the path link.
+ * The seconds are rounded up to the next integer.
+ */
+CREATE OR REPLACE FUNCTION estimateDuration(geo geometry, walking_speed NUMERIC) RETURNS INTERVAL AS
+$$
+DECLARE
+  duration interval;
+BEGIN
+  duration := (ST_Length(
+      ST_Transform($1, current_setting('export.PROJECTION')::int)::geography
+    )::NUMERIC / walking_speed)::INT;
+  RETURN duration;
+END
+$$
+LANGUAGE plpgsql IMMUTABLE STRICT;
```

Comment:
I would assume that we don't need the `duration` variable here. Instead we can directly return/select the result.
It can also be converted to SQL function instead of plpgsql like it is done here: https://github.com/OPENER-next/osm2vdv462/blob/2ec91f614745543bf2c7c4f9652071d041abe635/pipeline/stop_places/sql/stop_places.sql#L49-L55

I guess the last cast should then be `::INTERVAL` instead of `::INT` or `INTERVAL(...)`

naming convention `estimateDuration` > `estimate_duration`